nodes/standalone/src/service.rs

```diff
@@ -43,27 +43,22 @@ native_executor_instance!(
 type FullClient = sc_service::TFullClient<Block, RuntimeApi, Executor>;
 type FullBackend = sc_service::TFullBackend<Block>;
 type FullSelectChain = sc_consensus::LongestChain<FullBackend, Block>;
-type PartialConfig = sc_service::PartialComponents<
+type PartialComponents = sc_service::PartialComponents<
 	FullClient,
 	FullBackend,
 	FullSelectChain,
 	sp_consensus::DefaultImportQueue<Block, FullClient>,
 	sc_transaction_pool::FullPool<Block, FullClient>,
 	(
-		sc_consensus_aura::AuraBlockImport<
-			Block,
-			FullClient,
-			sc_finality_grandpa::GrandpaBlockImport<FullBackend, Block, FullClient, FullSelectChain>,
-			AuraPair,
-		>,
+		sc_finality_grandpa::GrandpaBlockImport<FullBackend, Block, FullClient, FullSelectChain>,
```

Comment:
Any reason why we switch to GrandPa from AuRa in the standalone runtime? Shouldn't we stick with the same consensus?

Reply:
We don't switch the consensus. We still use Aura+Grandpa. [Aura is just not needed for the block import anymore](https://github.com/paritytech/substrate/pull/8881).